templates/liar/templates.yaml

```diff
@@ -0,0 +1,78 @@
+dataset: liar
+templates:
+  331fec6f-bc64-4a81-b895-b1c5d5a61e7e: !Template
+    id: 331fec6f-bc64-4a81-b895-b1c5d5a61e7e
+    jinja: 'What was the party affiliation of the speaker of the following statement?
+
+      ===
```

Comment:
Can you include some prompts without this `===` or remove it from some of the prompts? I don't think this is a common feature of prompts, and it might teach the model to associate "===" with this dataset.

Reply:
Sure, I will remove `===` from some of the prompts.

Reply:
Updated! @craffel